### ISSUE TITLE
Hide add Transfer button if you only have one account

### ISSUE THREAD

**Issue by NPadrutt** [MoneyFox/MoneyFox#765]

**Comment by Squawk09** [comment]
Wow this is a really cool project that I can wrap my head around!

I would like to solve this problem but I have not contributed to any actual open source projects except for adding answers to a c++ book guide.

Do you have any advice on approaching this issue?  I have poked around the source code for the past two days and I have read the functions that are executed for the different button presses on the account menu.    

I don't understand how the actual GUI is drawn. I think the logic of hiding the transfer button is easy but i don't understand how to implement it.  Do you have any advice? 

Sorry if this is confusing,  I feel I am not knowledgeable enough to ask the question properly.

**Comment by NPadrutt** [comment]
First of all: Welcome! :)

Also, we have a slack group.  That often makes discussion a bit easier. Drop me mail with your mail at nino.padrutt@apply-solutions.ch and I send you an invite =)

I'm not sure how familiar you are with C# and XAML development. So I try to give you the basic information in case you didn't already know :)
We use a pattern called MVVM. You can find a lot of tutorials and easy descriptions on google (for example this one: http://www.codeproject.com/Articles/126249/MVVM-Pattern-in-WPF-A-Simple-Tutorial-for-Absolute)

In short: we try to separate view and logic. for the view we have the common .xaml views and for the logic ViewModels. These ViewModels can provide functions (Commands) or Information (Properties) who we bind to the UI via Bindings. Also for these Bindings we can make converter to help the system convert values (e.g. boolean to visibility).

Now, how I would takle this issue is to create a new property in the related ViewModel (this would be the MainViewModel) and bind this to the Visibility Property of the button.

Does that make any sense?
